qa/tasks/cephfs/test_snap_schedules.py

```diff
@@ -173,7 +175,7 @@ def remove_snapshots(self, dir_path):
     def test_non_existent_snap_schedule_list(self):
         """Test listing snap schedules on a non-existing filesystem path failure"""
         try:
-            self.fs_snap_schedule_cmd('list', f'path={TestSnapSchedules.TEST_DIRECTORY}')
+            self.fs_snap_schedule_cmd('list', path=f'{TestSnapSchedules.TEST_DIRECTORY}')
```

Comment:
nit
```suggestion
            self.fs_snap_schedule_cmd('list', path=TestSnapSchedules.TEST_DIRECTORY)
```

as `TestSnapSchedules.TEST_DIRECTORY` is always a `str`.

Reply:
fixed